modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -171,6 +172,16 @@ def show_local_breadcrumb
 
   private
 
+  def parse_drive_id
```

Comment:
🟡 just naming: ^^

`parse_drive_id` is invalid, if we are viewing this controller from a nc storage perspective. So, the question, can we extract the condition to the `before_action` (or is `@storage` not available there)? 

At least, we should rename the method here to something like: `prepare_update_params`, to be less OD specific.

Reply:
`parse_drive_id` is called as a `before_action` and `@storage` is available. But I don't understand the question here.

I agree on the naming. I'll change it.